Tests/XCUITests/HistoryTests.swift

```diff
@@ -38,45 +39,39 @@ class HistoryTests: BaseTestCase {
     }
 
     func testEmptyHistoryListFirstTime() {
-        navigator.performAction(Action.CloseURLBarOpen)
```

Comment:
I think we can't remove this... otherwise we stay in this screen with the 
![Simulator Screen Shot - iPhone 14 - 2022-10-20 at 10 08 50](https://user-images.githubusercontent.com/1897507/196894866-f70b8adb-edf4-454c-98d6-e690ee2a2d2d.png)
The [BrowserTabMenu](https://github.com/mozilla-mobile/firefox-ios/blob/8ff58f10802b3febc4c84d690b6f3bd1d0d13e6b/Tests/XCUITests/FxScreenGraph.swift#L938) button is hidden behind the keyboard and so we can't access to the menu to go to the `LibraryPanel_History` in the following L44

Reply:
When I run the test locally on my computer, the keyboard is hidden in the beginning by default without `Action.CloseURLBarOpen`. Is this another difference between Intel and M1?
![Simulator Screen Shot - iPhone 14 - 2022-10-20 at 10 22 30](https://user-images.githubusercontent.com/1740517/196975323-ed875e20-36bf-44b0-83c0-9b811cf9e712.png)

Reply:
At that point in the screenshots the test has not started yet or for some reason the keyboard is not shown on your side, they keyboard should appear as the url bar is focused. I'm running the tests on M1, that's not an M1 issue